server/CURRENT_VERSION

```diff
@@ -0,0 +1 @@
+2.14.0-beta.2
```

Comment:
Please don't check this in. For some reason, this is missing from the _.gitignore_ file; I am remedying this.

For local development, you can write "12345" to this file.

Reply:
1. Flake builds can only use files that are tracked in version control (as the build depends on this not-deterministic file)
2. I thought the version was used to fetch the CDN FE assets for the Console, is this not correct?

Reply:
Apologies, I thought I'd replied to your comments.

I suggest adding a `preConfigure` step that writes the file with the contents of the package version then.